### ISSUE TITLE
Question about usage.

### ISSUE THREAD

**Issue by lazakoa** [gorilla-co/odata-query#21]
Is there support for parsing a query, rewriting values in the AST, and generating an OData query string again?

**Comment by OliverHofkens** [comment]
No yet, but that does sound like a cool use case!

**Comment by OliverHofkens** [comment]
Happy to say this was implemented in #27 and will be released in v0.7.0.
This release is ready to be published, but PyPI is currently in maintenance mode.

It works as follows:
```python
from odata_query.roundtrip import AstToODataVisitor

transformer = AstToODataVisitor()
odata_string = transformer.visit(ast)
```

Example test cases can be found here:
https://github.com/gorilla-co/odata-query/blob/master/tests/integration/test_roundtrip.py

Thank you for this cool feature request!
